core/orch_test.go

```diff
@@ -1207,13 +1207,13 @@ func TestIsActive(t *testing.T) {
 	}
 	orch := NewOrchestrator(n, rm)
 
-	ok, err := orch.isActive(addr)
+	ok, err := orch.isPaymentEligible(addr)
```

Comment:
WDYT about adding an additional test for when `addr` is pending activation?

Reply:
Good idea, added in https://github.com/livepeer/go-livepeer/pull/2146/commits/9ae8d38313c948746d08d25d9e04113010a77d48